spec/supabase_csharp_v0.yml

```diff
@@ -0,0 +1,1669 @@
+  #           .listen((List<Map<String, dynamic>> data) {
+  #           // Do something awesome with the data
+  #         });
+  #         ```
+  #     - id: listening-to-a-specific-rows-within-a-table
+  #       name: Listening to a specific rows within a table
+  #       description: |
+  #         You can listen to individual rows using the format `{table}:{col}=eq.{val}` - where `{col}` is the column name, and `{val}` is the value which you want to match.
+  #         This syntax is the as how you can filter data in Realtime
+  #       code: |
+  #         ```dart
+  #         supabase.from('countries')
+  #           .stream(primaryKey: ['id'])
+  #           .eq('id', '120')
+  #           .listen((List<Map<String, dynamic>> data) {
+  #           // Do something awesome with the data
+  #         });
+  #         ```
+  #     - id: with-order
+  #       name: With `order()`
+  #       code: |
+  #         ```dart
+  #         supabase.from('countries')
+  #           .stream(primaryKey: ['id'])
+  #           .order('name', ascending: true)
+  #           .listen((List<Map<String, dynamic>> data) {
+  #           // Do something awesome with the data
+  #         });
+  #         ```
+  #     - id: with-limit
+  #       name: With `limit()`
+  #       code: |
+  #         ```dart
+  #         supabase.from('countries')
+  #           .stream(primaryKey: ['id'])
+  #           .order('name', ascending: true)
+  #           .limit(10)
+  #           .listen((List<Map<String, dynamic>> data) {
+  #           // Do something awesome with the data
+  #         });
+  #         ```
+  #     - id: using-stream-with-stream-builder
+  #       name: Using `stream()` with `StreamBuilder`
+  #       description: |
+  #         When using `stream()` with a `StreamBuilder` within your Flutter application, make sure to store your stream in a variable to prevent refetching upon rebuilding.
+  #       code: |
+  #         ```dart
+  #         final supabase = Supabase.instance.client;
+
+  #         class MyWidget extends StatefulWidget {
+  #           const MyWidget({Key? key}) : super(key: key);
+
+  #           @override
+  #           State<MyWidget> createState() => _MyWidgetState();
+  #         }
+
+  #         class _MyWidgetState extends State<MyWidget> {
+  #           // Persist the stream in a local variable to prevent refetching upon rebuilds
+  #           final _stream = supabase.from('countries').stream(primaryKey: ['id']);
+
+  #           @override
+  #           Widget build(BuildContext context) {
+  #             return StreamBuilder(
+  #               stream: _stream,
+  #               builder: (context, snapshot) {
+  #                 // Return your widget with the data from the snapshot
+  #               },
+  #             );
+  #           }
+  #         }
+  #         ```
+
+  - id: list-buckets
+    description: |
+      Retrieves the details of all Storage buckets within an existing product.
```

Comment:
You'll notice that "buckets" doesn't get output properly here. That's due to separate issue that we've fixed yesterday. You should get that after you merge master in.

<img width="756" alt="CleanShot 2023-01-24 at 12 34 49@2x" src="https://user-images.githubusercontent.com/105593/214344938-60f1c356-5fbb-4950-b4c5-cc844a89e73f.png">